script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -205,17 +207,20 @@ I accepted her words of concern with an open heart.
 [sha:53e2e7e45a8aedf543381f1fcd88f6fbd6fcb71f]{
 -- Page 16, Offset 1185.
 -- この七年間は本当に穏やかだった。それと同じように、自分のいた時間が有間の家の人たちにとって苦痛でなかったのなら、これほど喜ばしい事はない。
-The past seven years have been truly peaceful. As long as I hadn't caused trouble for the Arimas, I couldn't be happier.
+The past seven years have been truly peaceful for me.
+ And in the same way,
+ if the Arimas didn't feel bad with me around,
+ then I couldn't be happier.
 }
 [sha:1d22acf4ffa589592379e4f8ce0dd15088282d88]{
 -- Page 17, Offset 1186. Mods: @n.
 -- “七年もたてば人並みに健康な体に戻ります。
-'After seven years, I'm as healthy as a man can be.
+'If you give it seven years, the body is bound to return to its normal healthy ways.
```

Comment:
Prefer the original here. I really dislike 'body is bound to return to its normal healthy ways', sounds like he's spouting pretentious bullshit.